### DISCUSSION TITLE
New renderer on 18.2.0 of the Maps SDK for Android

### DISCUSSION THREAD

**Discussion by radekzz** [react-native-maps/react-native-maps#4979] [Q&A]
Google says  _"To provide an improved user experience and app performance, apps built with the Maps SDK for Android will be automatically updated to the [latest renderer](https://notifications.google.com/g/p/ANiao5pxaNa-3BacL9wv-mQmeE4Ry2ywI78_BLUhv1NVJxU1Op3IKX9tZ1Q-lUCHezM9mh0g8YRvS2DflUyRFKpp7Y0BO-RQtuAV0hjdWUo5yi4jANRxnLe0NTaCcQsLTi05StxCMbjPEn2FBLR5gW1XCN8G2oEvujxFBoxg0UBC1cbSSe4uijE1Gd9HqlPKRR5WaxefgCiMlp5PDuVOpm3-6GJD_2xpZUAQdWEpv6Zq3ah3GCrMvH_BwZKlRYaP-lvnCA) beginning March 4, 2024. The legacy renderer is scheduled for decommissioning on March 3, 2025."_

I have a list of my apps that are using a legacy renderer so i would like to know if react-native-maps is prepared for this change and how can i change / test the new renderer.

Did you guys face it already?

**Comment by orfreesbe** [comment]
Same, i got an email from google that i have to update my SDK.
Any actions that we need to take to update manually the SDK?
